src/styles/blocks/card.scss

```
@@ -0,0 +1,90 @@
+.card {
+  width: $cell-size;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  margin: 50px 48px 46px 0;
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+  &:nth-last-child(-n + 4) {
```

add blanks between selectors
```suggestion
  margin: 50px 48px 46px 0;
  
  &:nth-child(4n) {
    margin-right: 0;
  }
  
  &:nth-last-child(-n + 4) {
```